lib/oeqa/selftest/cases/updater_qemux86_64_non_ostree.py

```diff
@@ -0,0 +1,47 @@
+import updater_qemux86_64
+from updater_qemux86_64 import GeneralTests, AktualizrToolsTests, SharedCredProvTests, \
+    ManualControlTests, DeviceCredProvTests, DeviceCredProvHsmTests, ResourceControlTests, \
+    DeviceCredProvHsmTests, IpSecondaryTests
+
+
+updater_qemux86_64.UBOOT_ENABLE = 'no'
+
+
+class GeneralTestsNonOSTree(GeneralTests):
+    pass
+
+
+class AktualizrToolsTestsNonOSTree(AktualizrToolsTests):
+    pass
+
+
+class SharedCredProvTestsNonOSTree(SharedCredProvTests):
+    pass
+
+
+class ManualControlTestsNonOSTree(ManualControlTests):
+    pass
+
+
+class DeviceCredProvTestsNonOSTree(DeviceCredProvTests):
+
+    def setUpLocal(self):
+        self.append_config('IMAGE_INSTALL_remove += " aktualizr-shared-prov"')
+        self.append_config('IMAGE_INSTALL_append += " aktualizr-device-prov"')
+        super().setUpLocal()
+
+
+class DeviceCredProvHsmTestsNonOSTree(DeviceCredProvHsmTests):
+
+    def setUpLocal(self):
+        self.append_config('IMAGE_INSTALL_remove += " aktualizr-shared-prov"')
+        self.append_config('IMAGE_INSTALL_append += " aktualizr-device-prov-hsm"')
+        super().setUpLocal()
+
+
+class IpSecondaryTestsNonOSTree(IpSecondaryTests):
+    pass
+
+
+class ResourceControlTestsNonOSTree(ResourceControlTests):
+    pass
```

Comment:
I'm pretty confused by the goal here. I don't think you need to make a copy of every oe-selftest for the non-ostree case. One test case that sets up the environment, builds an image, and boots it should be enough.

Reply:
If we want to reuse existing code(test cases) which is common, and run tests independently for `ostree` and `non-ostree `we have to do this, I didn't look into OESelftestTestCase implementation but  I think it works in the way like looking all classes which inherit OESelftestTestCase and runs all member functions with the prefix test_* so when we run `oe-selftest `we pass a python module(file) and if we want reuse tests from another python module(file) we have to do something like that
https://github.com/advancedtelematic/meta-updater/blob/fed844e7b6b0ffdf52346aaf23ed95689018c401/lib/oeqa/selftest/cases/updater_qemux86_64_non_ostree.py#L10

`oe-selftest` doesn't look into imported modules): to make them visible for `oe-selftes` we do this hack